File: qlty-cli/tests/lang/kotlin/basic.in/lines.kt
fun f1() {
    val sourceFile = File.createTempFile("ruby", ".kt")
    sourceFile.writeText("foo(*args)")
    val tree = sourceFile.parse()

    bar()
}

// Foo
fun f2() {
    bar() // does not count as comment line
}

// multi-line comment
/*
line1
line2

line4
*/
fun f3() {
    bar()
}

File: qlty-cli/tests/lang/rust/basic.in/lines.rs
fn f1() {
    let source_file = File::from_string("ruby", "foo(*args)");
    let tree = source_file.parse();

    bar();
}

// Foo
fn f2() {
    bar(); // does not count as comment line
}

// multi-line comment
/*
line1
line2

line4
*/
fn f3() {
    bar();
}
